### PR TITLE
[LPTOCPCI-315] Add config for Kiali for ROSA hypershift cluster

### PR DESCRIPTION
**Pull request by smpsgill**
Adding this initial config for Kiali operator install in a ROSA hypershift cluster.

[Git repo](https://github.com/kiali/kiali-operator) for Kiali operator is meant to be used within the [kiali/kiali](https://github.com/kiali/kiali) parent repo which contains the tests

### CODE REVIEW

File: ci-operator/config/kiali/kiali/kiali-kiali-master__hypershift-4.13.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "osd-*", "failure_type": "all", "classification": "User Setup", "jira_project": "LPTOCPCI"},
+          {"step": "rosa-*", "failure_type": "all", "classification":  "Cluster Creation", "jira_project": "LPTOCPCI"}
```
Comment (ascerra) [inline ci-operator/config/kiali/kiali/kiali-kiali-master__hypershift-4.13.yaml:52]: You could also add the following to this firewatch config:
```         
 {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI"},
{"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Cluster Health Check", "jira_project": "LPTOCPCI"}
```

File: ci-operator/config/kiali/kiali/kiali-kiali-master__hypershift-4.13.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-rosa-hypershift-lp-interop-poc
+  cron: 0 6 * * 1
```
Comment (ascerra) [inline ci-operator/config/kiali/kiali/kiali-kiali-master__hypershift-4.13.yaml:42]:
```suggestion
  cron: 0 10 * * 1
```

File: ci-operator/config/kiali/kiali/kiali-kiali-master__hypershift-4.13.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-rosa-hypershift-lp-interop-poc
```
Comment (ascerra) [inline ci-operator/config/kiali/kiali/kiali-kiali-master__hypershift-4.13.yaml:41]:
```suggestion
- as: aws-lp-rosa-hypershift
```

File: ci-operator/config/kiali/kiali/kiali-kiali-master__hypershift-4.13.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  cron: 0 6 * * 1
+  remote_api: true
```
Comment (ascerra) [inline ci-operator/config/kiali/kiali/kiali-kiali-master__hypershift-4.13.yaml:43]: We can remove this line

**Comment by ascerra** [comment]
Waiting on the bots to be added to the kiali github org for this test is not needed, we can just use the config/rhpit/interop-tests directory as nothing from the operators repo is needed to install the operator and run the hypershift health check.

**Review by calebevans — APPROVED** [review]
/lgtm

**Comment by jwendell** [comment]
Hi @Simranpal . Any reason why maistra group was included as approvers/reviewers?

**Comment by Simranpal** [comment]
@jwendell it should be the kiali team. I will check if there is a group created for them, and create a PR updating the reviewers

**Comment by jwendell** [comment]
@Simranpal Thank you!

**Pull request merged**